aspnetcore/mvc/views/tag-helpers/th-components/samples/RazorPagesSample/Pages/Privacy.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@page
+@model PrivacyModel
+@{
+    ViewData["Title"] = "Privacy Policy";
+}
+<h2>@ViewData["Title"]</h2>
+
+<p>Use this page to detail your site's privacy policy.</p>
```

Comment:
CRLF to make GH happy? ... I recall that you had some trouble before with code examples, but I haven't had a problem yet going ahead with CRLFs in these cases.